src/test/scala/is/hail/methods/LinearRegressionSuite.scala

```diff
@@ -135,9 +135,9 @@ class LinearRegressionSuite extends SparkSuite {
     def assertEmpty(q: Querier, v: Variant) =
       assert(q(annotationMap(v)) == null)
 
-    val gt0 = Genotype.phredToBiallelicDosageGT(Array(0, 20, 100))
-    val gt1 = Genotype.phredToBiallelicDosageGT(Array(20, 0, 100))
-    val gt2 = Genotype.phredToBiallelicDosageGT(Array(20, 100, 0))
+    val gt0 = Genotype.phredToBiallelicDosageGenotype(Array(0, 20, 100))
+    val gt1 = Genotype.phredToBiallelicDosageGenotype(Array(20, 0, 100))
+    val gt2 = Genotype.phredToBiallelicDosageGenotype(Array(20, 100, 0))
 
     assert(D_==(gt0, 0.009900990296049406))
     assert(D_==(gt1, 0.9900990100009803))
```

Comment:
AFAICT, these six lines have no dependencies on the rest of this test, so they shouldn't be here. It sound like they belong in `GenotypeSuite`.